src/service.js

```diff
@@ -28,6 +28,26 @@ module.exports = function(mixinOptions) {
 	});
 
 	const serviceSchema = {
+		actions: {
+			ws: {
+				visibility: "private",
+				tracing: {
+					tags: {
+						params: [
+							"socket.upgradeReq.url"
+						]
+					},
+					spanName: ctx => `UPGRADE ${ctx.params.socket.upgradeReq}`
+				},
+				handler(ctx) {
+					const { socket, connectionParams } = ctx.params
```

Comment:
Can you run a lint against this file?  I'm seeing some errors reported (e.g. no semi at the end of this line).

Reply:
Oh sorry I forget to check that before push, done.